repo/packages/F/flink/0/marathon.json.mustache

```diff
@@ -3,7 +3,8 @@
   "cpus": {{app-master.cpus}},
   "mem": {{app-master.memory}},
   "env": {
-    "DCOS_SERVICE_NAME": "{{service.name}}"
+    "DCOS_SERVICE_NAME": "{{service.name}}",
+    "FLINK_LOG_DIR": "/mnt/mesos/sandbox"
```

Comment:
That's my question if you check the comment above I saw that it is used inside the bash command in the docker not sure if it would work there. I will change it.